bigtable/tests/mutations_integration_test.cc

```diff
@@ -0,0 +1,555 @@
+                                std::string column_id, std::int64_t time_end) {
+    bigtable::SingleRowMutation mutation(row_key);
+    mutation.emplace_back(
+        bigtable::DeleteFromColumnEndingAt(column_family, column_id, time_end));
+
+    bigtable::BulkMutation bulk;
+    bulk.emplace_back(std::move(mutation));
+    table.BulkApply(std::move(bulk));
+  }
+
+  /**
+   * Delete the records from table for the specified family and column with
+   * timestamp starting from time_begin.
+   * time_begin: This parameter is inclusive i.e. timestamp >= time_begin
+   * time_end  : This parameter is not inclusive. i.e. timestamp < time_end
+   */
+  void DeleteFromColumn(bigtable::Table& table, std::string row_key,
+                        std::string column_family, std::string column_id,
+                        std::int64_t time_begin, std::int64_t time_end) {
+    bigtable::SingleRowMutation mutation(row_key);
+    mutation.emplace_back(bigtable::DeleteFromColumn(column_family, column_id,
+                                                     time_begin, time_end));
+
+    bigtable::BulkMutation bulk;
+    bulk.emplace_back(std::move(mutation));
+    table.BulkApply(std::move(bulk));
+  }
+
+  /**
+   * Delete records from a specific row_key and specific column_family
+   */
+  void DeleteFromFamily(bigtable::Table& table, std::string row_key,
+                        std::string column_family) {
+    bigtable::SingleRowMutation mutation(row_key);
+    mutation.emplace_back(bigtable::DeleteFromFamily(column_family));
+
+    bigtable::BulkMutation bulk;
+    bulk.emplace_back(std::move(mutation));
+    table.BulkApply(std::move(bulk));
+  }
+
+  /**
+   * Delete records from a specific row key
+   */
+  void DeleteFromRow(bigtable::Table& table, std::string row_key) {
+    bigtable::SingleRowMutation mutation(row_key);
+    mutation.emplace_back(bigtable::DeleteFromRow());
+
+    bigtable::BulkMutation bulk;
+    bulk.emplace_back(std::move(mutation));
+    table.BulkApply(std::move(bulk));
+  }
+};
+
+}  // namespace anonymus
+
+/**
+ *  Check if the values inserted by SetCell are correctly inserted into
+ *  Cloud Bigtable
+ */
+TEST_F(MutationIntegrationTest, SetCellTest) {
+  std::string const table_name = "table-setcell";
+
+  auto table = CreateTable(table_name, table_config);
+  // Create a vector of cells which will be inserted into bigtable
+  std::string const row_key = "SetCellRowKey";
+  std::vector<bigtable::Cell> created_cells{
+      {row_key, column_family1, "column_id1", 0, "v-c-0-0", {}},
+      {row_key, column_family1, "column_id1", 1000, "v-c-0-1", {}},
+      {row_key, column_family1, "column_id1", 2000, "v-c-0-2", {}},
+      {row_key, column_family2, "column_id2", 0, "v-c0-0-0", {}},
+      {row_key, column_family2, "column_id3", 1000, "v-c1-0-1", {}},
+      {row_key, column_family3, "column_id1", 2000, "v-c1-0-2", {}},
+  };
+
+  CreateCells(*table, created_cells);
```

Comment:
nit: in general I would remove the extra blank lines here (and other tests), i.e.:

```C++
CreateCells(*table, created_cells);
auto actual_cells = ReadRows(*table, bigtable::Filter::PassAllFilter());
DeleteTable(table_name);
CheckEqualUnordered(created_cells, actual_cells);
```

Reply:
PTAL